src/Azos/Data/attrs/FieldAttribute.cs

```diff
@@ -521,44 +494,44 @@ public override bool Equals(object obj)
       if (!base.Equals(other)) return false; //target metadataContent description
 
       var equ =
-          this.StoreFlag   == other.StoreFlag &&
-          this.BackendName.EqualsOrdSenseCase(other.BackendName) &&
-          this.BackendType.EqualsOrdSenseCase(other.BackendType) &&
-          this.Key         == other.Key &&
-          this.Kind        == other.Kind &&
-          this.Required    == other.Required &&
-          this.Visible     == other.Visible &&
+          StoreFlag   == other.StoreFlag &&
```

Comment:
I would keep `.this` prefixes in Equals method which compare field-by field for clarity